cmd/kops/create_cluster.go

```diff
@@ -59,6 +59,8 @@ type CreateClusterOptions struct {
 	MasterSize           string
 	MasterCount          int32
 	NodeCount            int32
+	MasterVolumeSize     int32
```

Comment:
I think MB is the most flexible choice, personally I usually wouldn't want volumes over a 100MB on my master.

I don't see a use for sub-MB volume size granularities outside very fringe cases.